homeassistant/components/apache_kafka/__init__.py

```diff
@@ -130,6 +130,12 @@ def _encode_event(self, event: Event[EventStateChangedData]) -> bytes | None:
             "utf-8"
         )
 
+    def _get_key(self, event: Event[EventStateChangedData]) -> bytes | None:
+        """Get the entity_id to use as key."""
+        key = event.data["entity_id"]
+
+        return key.encode("utf-8")
```

Comment:
Any reason not to consolidate this?

```suggestion
        return event.data["entity_id"].encode("utf-8")
```

Reply:
Consolidated and inlined.